src/main/java/uk/gov/companieshouse/company/profile/config/ApplicationConfig.java

```diff
@@ -21,6 +23,7 @@
 import org.springframework.security.web.authentication.www.BasicAuthenticationFilter;
 import uk.gov.companieshouse.api.company.Data;
 import uk.gov.companieshouse.api.converter.EnumWriteConverter;
+import uk.gov.companieshouse.api.model.CompanyProfileDocument;
```

Comment:
Are we adding unused import statements?

Reply:
Yep my bad missed that. Already merged but there is another pr to remove them https://github.com/companieshouse/company-profile-api/pull/167